backend/app/api/routes/login_modern.py

```
@@ -0,0 +1,333 @@
+        "cache_operations_ms": 0,
+        "database_query_ms": 0,
+    }
+
+    try:
+        # Step 1: 数据库查询用户
+        db_start = time.time()
+
+        # 优化的查询 - 使用新索引
+        statement = select(User).where(
+            User.email == form_data.username,
+            User.is_active == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)